### DISCUSSION TITLE
robots.txt manual editing

### DISCUSSION THREAD

**Discussion by crztssr** [requarks/wiki#6221] [Help / Questions]
Hello! Thanks for a great product! 
Unfortunately I'm having a little issue with my domain - as I found through the  webarchive, previous owner used the domain for scam purposes and was selling fake diplomas and other similar staff.
So through the wemaster console of one of search engines I'm observing the links which existed on the domain in 2018th, but not presented on by wiki.js site currently.
Search engine's support guys told me that the only way to avoid seeing these pages in webmaster console is to add a Dissallow rule to robots.txt for this specific URLs. 

I don't want to dissalow all pages of the wiki. Is there any ways to add custom Dissallow rules to robots.txt?

P.S. Yeah, I know that I can try to setup a webserver with robots.txt and setup nginx redirection for mywikijsdomain.com/robots.txt to it. But I would like to do it with wiki.js only if it is possible.
P.P.S. I have an installation without docker, using PostgreSQL 15 as a db.

**Comment by Philipinho** [comment]
There is unfortunately no way to update it directly from Wiki.js. You may want to consider the NGINX option again.
